feature-detects/mediastream.js

```diff
@@ -0,0 +1,13 @@
+define(['Modernizr'], function (Modernizr) {
```

Comment:
Hi! Every Modernizr file is required to have metadata (it serves managing and other purposes), you may [check the schema of the documentation in the docs (linked)](https://github.com/Modernizr/Modernizr/blob/master/HOW_TO_WRITE_FEATURE_DETECTS.md#metadata)

Reply:
Metadata still missing (just click the link and follow the guide)

Reply:
Okay, you've added the testing, but the metadata is still missing (hence, the failed builds). The metadata goes inside the mediastream.js file, at the top, you may check any of the other feature-detects to see an example.
That being said, sorry because in my previous comment I sent you to the testing section instead of the metadata one (my bad). Testing still needed adding so it wasn't worthless your job :)
Here is the correct link this time:
https://github.com/Modernizr/Modernizr/blob/master/HOW_TO_WRITE_FEATURE_DETECTS.md#metadata

Come on! You are almost there, almost nothing left!